test/prod/compound/TestUpgradeLens.t.sol

```diff
@@ -124,7 +124,7 @@ contract TestUpgradeLens is TestSetup {
                 assertApproxEqAbs(
                     lens.getCurrentUserSupplyRatePerBlock(supplyMarket.poolToken, address(user)),
                     expectedSupplyRate,
-                    1,
+                    1e16,
```

Comment:
this very large no?

Reply:
Yes, but it's the same as the test in aave-v2. I'll look into it

Reply:
I investigated and found that:
- the number of matching iterations is correctly set to 1
- the difference comes from the pool rates (virtual vs actual) and is of similar size
- the difference is way lower to what we had before (so I decreased the deltas to 1e8) because of the fix to the borrow amount being able to include reserves

For these reasons, I'm marking this comment as resolved